site/src/pages/WorkspaceBuildPage/WorkspaceBuildPageView.tsx

```diff
@@ -308,24 +307,20 @@ type AgentLogsContentProps = {
 };
 
 const AgentLogsContent: FC<AgentLogsContentProps> = ({ agent }) => {
-	const logs = useAgentLogs(agent, true);
-
-	if (!logs) {
-		return <Loader />;
-	}
-
+	const logs = useAgentLogs({ agentId: agent.id });
 	return (
 		<AgentLogs
+			overflowed={false}
```

Comment:
is this actually correct? it seems like an overflow would be just as possible here